### PR TITLE
remove fat arrows.

### PR DESCRIPTION
**Pull request by paulirish**
sad. but necessary.

### CODE REVIEW

File: app/manifest.json [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+  },
+  "background": {
+    "scripts": [
+      "scripts/chromereload.js"
+    ]
+  },
+  "permissions": [
+    "clipboardWrite"
+  ]
+}
```
Comment (arthurvr) [inline app/manifest.json]: Why these changes?

**Comment by arthurvr** [comment]
Side: feel free to commit to master directly :wink:

**Comment by arthurvr** [comment]
Fix #15

File: app/manifest.json (hunk 1 above)
Comment (paulirish) [inline app/manifest.json]: `grunt build` is a jerk and did that. and then i got lazy because lack of sleep
